modules/codemod/lib/v5/spec/removeButtonEnums.spec.ts

```diff
@@ -0,0 +1,390 @@
+import {expectTransformFactory} from './expectTransformFactory';
+import transform from '..';
+
+const expectTransform = expectTransformFactory(transform);
```

Comment:
The `expectTransformFactory` seems to be just enough abstraction to make tests easier to write and understand. The signature of `expectTransform` is `input, output, options?`.